tests/numpy/array1.py

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
These were quite old and are more “drafts” or “examples”. Is there a place to put them? Otherwise I will move to another repository for me to play. Then “wurzel” will also be gone ;)

Reply:
outsourced to my own example repository for now